std_msgs/Header header

uint16 service_id
uint16 method_id
uint32 data_length
uint16 client_id
uint16 session_id
uint8 protocol_version
uint8 interface_version
uint8 message_type
uint8 return_code

uint64 crc
uint32 length
uint32 sqc
uint32 data_id
uint32 timestamp_nanoseconds
uint32 timestamp_seconds
uint8 timestamp_sync_status
uint32 event_data_qualifier
uint8 extended_qualifier
uint8 object_list_num_of_objects
Object[] objects